src/main/resources/templates/semesters.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorator="layout">
+<head>
+    <meta charset="UTF-8">
+    <title>Semesters</title>
+</head>
+<body>
+<div layout:fragment="content">
+    <h1>Semesters</h1>
+    <div class="container">
+        <table>
+            <tr>
+                <th>Season</th>
+                <th>Year</th>
+                <th>Information</th>
+            </tr>
+            <tr th:each="semester : ${semesters}">
+                <td th:text="${semester.season}"></td>
+                <td th:text="${semester.year}"></td>
+                <td><a class="badge badge-info"
+                       th:href="@{/semester/{id}(id=${semester.id})}">view
+                </a></td>
+            </tr>
+            <tr>
+                <a class="badge badge-primary" th:href=
```

Comment:
change this line to 
`<a class="badge badge-primary" href="/semesters/new">Create New Semester</a>`